spec/features/libraries_spec.rb

```diff
@@ -1,6 +1,12 @@
 require 'rails_helper'
 
 describe "Libraries" do
+  include_context 'login admin user'
+  
+  before(:each) do
+    @library = Library.create(name: "Lehman", code: "lehman")
```

Comment:
I just realized that we don't have Factory Girl in this project, yet. I'll add it to my branch and we can convert these to factories later on (I totally thought we already had the gem installed).